examples/starrocks/deploy_a_starrocks_cluster_with_fe_proxy.yaml

```diff
@@ -0,0 +1,53 @@
+# This manifest deploys a StarRocks cluster with FE, BE, FE Proxy
+# To load data from outside the Kubernetes cluster into StarRocks deployed within the Kubernetes
+# cluster, you can deploy the FE Proxy component. So that External data import tools, like flink,
+# can use the STREAM LOAD syntax to import data into StarRocks clusters.
+# see https://docs.starrocks.io/en-us/latest/loading/Flink-connector-starrocks for more information
+# about how to use flink to import data into StarRocks.
+
+apiVersion: starrocks.com/v1
+kind: StarRocksCluster
+metadata:
+  name: starrockscluster-sample
+spec:
+  starRocksFeSpec:
+    image: starrocks/fe-ubuntu:latest
+    replicas: 3
+    limits:
+      cpu: 8
+      memory: 16Gi
+    requests:
+      cpu: 8
+      memory: 16Gi
+    service:
+      type: NodePort   # export fe service
+      ports:
+      - name: query
+        nodePort: 32755
+        port: 9030
+        containerPort: 9030
+  starRocksBeSpec:
+    image: starrocks/be-ubuntu:latest
+    replicas: 3
+    limits:
+      cpu: 16
+      memory: 64Gi
+    requests:
+      cpu: 16
+      memory: 64Gi
+  starRocksFeProxySpec:
+    replicas: 1
+    limits:
+      cpu: 1
+      memory: 2Gi
+    requests:
+      cpu: 1
+      memory: 2Gi
+    service:
+      type: NodePort   # export fe proxy service
+      ports:
+        - containerPort: 8080
+          name: http-port
+          nodePort: 30180   # The range of valid ports is 30000-32767
+          port: 8080
+    resolver: "kube-dns.kube-system.svc.cluster.local"  # this is the default dns server.
```

Comment:
is this line always required?

Reply:
no, this is the default value if `resolver:""`